code/modules/mob/living/carbon/carbon.dm

```diff
@@ -273,6 +273,25 @@
 
 	if(!item) return
 
+	if(istype(item, /obj/item/stack/throwing_knife))
+		var/obj/item/stack/throwing_knife/V = item
+		var/ROB_throwing_damage = src.stats.getStat(STAT_ROB)
```

Comment:
Doesn't it work without `src.`?

Reply:
not really well, best to be sure

Reply:
There is `var/robust = stats.getStat(STAT_ROB)` in the same proc.

Reply:
well now there is no src.